apps/dashboard/src/components/icons/onboarding-arrow-left.tsx

```diff
@@ -0,0 +1,13 @@
+export function OnboardingArrowLeft(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
Not needed, please use chevron.

Reply:
This one is the hand written onboarding arrow, not a regular chevron icon. 

![CleanShot 2024-11-26 at 10 32 50@2x](https://github.com/user-attachments/assets/c4c56c5b-782a-4bdf-ae03-2fd725b5c8c9)

Reply:
ahh ok, sorry